saq/utils.py

```diff
@@ -16,3 +18,26 @@ def millis(s):
 
 def seconds(ms):
     return ms / 1000
+
+
+def exponential_backoff(
+    attempts: int,
```

Comment:
sorry for this nit but the types here aren't consistent with the rest of the codebase, can you remove?

Reply:
Ah, whoops. I had copy and pasted this from something else I wrote and left these in.